### PR TITLE
init command enhancements to enable running multiple instances of KDK…

### PR DESCRIPTION
**Pull request by rtluckie**
init command enhancements to enable running multiple instances of KDK container

- allow disparate configs
- reuse common ssh keypair
- add flags to allow user control of init config params
- rename `config` persistent flag -> `name`
- fix destroy cmd to support multiple KDKs
- update README
- gofmt changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,13 +170,11 @@ kdk ssh
 kdk destroy
 ```
 
-
 ## Customization
 
 * **NOTE:**  By default, the `KDK` uses a set of opinionated dotfiles. 
 To customize, fork [this](https://github.com/cisco-sso/yadm-dotfiles) repo, make changes, and update `~/.kdk/config.yaml` to reference customized fork.
```
Comment (dcwangmit01) [inline README.md]: It's no longer ~/.kdk/config.yaml but `~/.kdk/<kdk-name>/config.yaml`

File: cmd/init.go [hunk 2]
```diff
@@ -20,19 +20,34 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
```
Comment (dcwangmit01) [inline cmd/init.go]: MUST: Please check up on this.  Can you put these directly into an AppConfig Struct, and then pass that AppConfig struct directly to the marshal function in root.go

File: internal/app/kdk/init.go [hunk 3]
```diff
@@ -15,52 +15,60 @@
...
-	ConfigName       string
-	ConfigPath       string
-	Verbose          bool
-	KeypairDir       string
-	PrivateKeyPath   string
-	PublicKeyPath    string
-	DockerClient     *client.Client
-	Ctx              context.Context
-	KdkConfig        *kdkConfig
+	ConfigDir      string
```
Comment (dcwangmit01) [inline internal/app/kdk/init.go]: I think these can go in an AppConfig struct.  We can pair-program.

File: internal/app/kdk/init.go [hunk 4]
```diff
@@ -119,29 +127,29 @@ func InitKdkConfig(logger logrus.Entry) error {
 
 	// Define Additional volume bindings
 	for {
-		prompt := promptui.Prompt {
-			Label: "Would you like to mount additional docker host directories into the KDK? [y/n]",
-			Default: "",
+		prompt := promptui.Prompt{
+			Label:     "Would you like to mount additional docker host directories into the KDK? [y/n]",
```
Comment (dcwangmit01) [inline internal/app/kdk/init.go]: Did you run this through an autoformatter?  Can you put it in the Makefile?

Comment (rtluckie) [reply internal/app/kdk/init.go]: `gofmt -s -w cmd internal main.go`

File: README.md (hunk 1 above)
Comment (rtluckie) [reply README.md]: fixed

File: cmd/init.go (hunk 2 above)
Comment (dcwangmit01) [reply cmd/init.go]: Chatted with Ryan.  He's going to go back and refactor the init anyways.

**Pull request merged**